frontend/src/containers/pages/AuthoringTool/CanvasSideBar/SceneSettings.jsx

```diff
@@ -70,6 +128,54 @@ export default function SceneSettings() {
               shrink: !!currentScene?.time,
             }}
           />
+          <FormControl fullWidth>
+            <CustomInputLabel>Scene Role(s)</CustomInputLabel>
+            <Select
```

Comment:
Could we not have the checkbox close every time we select something? Remember that we can have multiple roles access the same scene.

See [MUI Select - Checkmarks](https://mui.com/material-ui/react-select/#checkmarks) (pay close attention to the `multiple`)